### ISSUE TITLE
Why there are multiple pvc？

### ISSUE THREAD

**Issue by wajika** [elastic/cloud-on-k8s#3008]
```
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: elastic-local-volume
provisioner: kubernetes.io/no-provisioner
volumeBindingMode: WaitForFirstConsumer
---
apiVersion: v1
kind: PersistentVolume
metadata:
  name: es-data-pv-1
spec:
  capacity:
    storage: 10Gi
  accessModes:
    - ReadWriteOnce
  persistentVolumeReclaimPolicy: Recycle
  storageClassName: elastic-local-volume
  local:
    path: /data/elastic-pv1/
  nodeAffinity:
    required:
      nodeSelectorTerms:
        - matchExpressions:
            - key: kubernetes.io/hostname
              operator: In
              values:
                - 'node-2'
---
apiVersion: v1
kind: PersistentVolume
metadata:
  name: es-data-pv-2
spec:
  capacity:
    storage: 10Gi
  accessModes:
    - ReadWriteOnce
  persistentVolumeReclaimPolicy: Recycle
  storageClassName: elastic-local-volume
  local:
    path: /data/elastic-pv2/
  nodeAffinity:
    required:
      nodeSelectorTerms:
        - matchExpressions:
            - key: kubernetes.io/hostname
              operator: In
              values:
                - 'node-2'




apiVersion: elasticsearch.k8s.elastic.co/v1
kind: Elasticsearch
metadata:
  name: elastic
spec:
  version: 7.6.2
  image: ccr.ccs.tencentyun.com/elk-oss/elasticsearch:7.6.2
  nodeSets:
  - name: elastic-master
    count: 2
    config:
      node.master: true
      node.data: false
      node.ingest: false
      node.ml: false
      node.store.allow_mmap: false
      index.store.type: niofs
      cluster.remote.connect: false
    podTemplate:
      spec:
        containers:
        - name: elasticsearch
          resources:
            limits:
              memory: 4Gi
              cpu: 2
          env:
          - name: ES_JAVA_OPTS
            value: "-Xms2g -Xmx4g"
    volumeClaimTemplates:
    - metadata:
        name: elastic-master-data
      spec:
        accessModes:
        - ReadWriteOnce
        resources:
          requests:
            storage: 2Gi
        storageClassName: elastic-local-volume
```

 

![image](https://user-images.githubusercontent.com/17058349/80565486-28ba2e00-8a23-11ea-8cc9-2963afc36066.png)

elasticsearch-data-elastic-es-elastic-master-0
elasticsearch-data-elastic-es-elastic-master-1
How does "elasticsearch-data" appear?

Why does elasticsearch pod show "pod has unbound immediate PersistentVolumeClaims"

**Comment by thbkrkr** [comment]
> Why there are multiple PVC?

ECK requests one PVC per Elasticsearch node.

You asked for an Elasticsearch cluster with **2** master nodes, you got **2** PVC.

> How does "elasticsearch-data" appear?

ECK always creates an `elasticsearch-data` PV per Elasticsearch node.

To increase the capacity of this volume, [the name of the volume claim must always be **elasticsearch-data**](https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-volume-claim-templates.html).

> Why does elasticsearch pod show "pod has unbound immediate PersistentVolumeClaims"

This happens when there is no PersistentVolume to satisfy the claim. 

Depending on your Kubernetes platform / Cloud provider, the problem can come from many different sources: [a bad pod anti-affinity rule](https://github.com/elastic/cloud-on-k8s/issues/920), [an issue with minikube](https://github.com/elastic/cloud-on-k8s/issues/2209), a quota reached at your cloud provider, not enough disk space on your machine if you are in local, etc.

What's your Kubernetes platform/version?

**Comment by wajika** [comment]
@thbkrkr Thank you for your reply.

I'm sorry I didn't make the meaning strap clear.

I mean I declared two nodes and then provided two pv. Why do I need to provide the "elasticsearch-data" pv again? In the yaml file, I did not define it

Why the "elasticsearch-data" information of the stroageclass column is empty?  I defined the stroageclass in yaml
![image](https://user-images.githubusercontent.com/17058349/80662549-1a701e80-8ac4-11ea-9e9e-96c243092d18.png)

If I want elasticsearch to work properly, do I need to declare two volumes?

Regarding the version of kubernetes, I also encountered a problem. When creating elasticsearch yaml, it succeeded in v1.16.8, but in v1.14.4, it prompts timeout, I do n’t know why

```
kubectl create -f elasticsearch.yaml
Error from server (Timeout): error when creating "elasticsearch.yaml": Timeout: request did not complete within requested timeout 30s
```
